shared/db/create-query.js

```diff
@@ -0,0 +1,95 @@
+/* @flow
+ *
+ * Create a RethinkDB query and cache it's result in a TagCache
+ *
+ * Usage:
+ *
+ * const getThreadById = createReadQuery({
```

Comment:
I think this documentation needs to be updated with the new callback syntax, right?